frontend/global-styles/index.scss

```diff
@@ -30,18 +30,6 @@ body {
   background-color: $dark;
 }
 
-#__next {
-  height: 100vh;
-}
-
-html {
-  height: 100%;
-}
-
-body {
-  min-height: 100%;
-}
```

Comment:
Why did we add this in the first place? Does something break when removing this?

Blame only shows the initial squash commit from the original private repo for the frontend.

Reply:
Because we needed it but now we dont.

Seems like no.

Reply:
IIRC this is either from the beginning of the bootstrap 3 usage when we recreated the first layouts.. or from the old HD code. anyway. I checked the pages and they still look okay